src/ext.rs

```diff
@@ -20,62 +20,89 @@
 
 //! Provides the `TransportExt` trait.
```

Comment:
Why rename `transport_ext` to `ext`? If we want more extension traits for other things, I think we'd put them in a different module.

Reply:
Sorry, it was a left-over from some experimentation. Reverted back to `transport_ext`.